test/runtests.jl

```diff
@@ -1,6 +1,10 @@
 using FactorizationMachines
 using Base.Test
 
-include("test_fm_readlibsvm.jl")
+include("test_read_libsvm.jl")
 include("test_fm.jl")
```

Comment:
There is still `fmTrain`. Test is successful because the exception is expected. However, the different one is given ;-)